contracts/root/withdrawManager/WithdrawManagerStorage.sol

```diff
@@ -4,15 +4,24 @@ import { ProxyStorage } from "../../common/misc/ProxyStorage.sol";
 import { Registry } from "../../common/Registry.sol";
 import { RootChain } from "../RootChain.sol";
 
+contract ExitsDataStructure {
+  struct Input {
+    address signer;
```

Comment:
are we planning to add more data in `Input`? which one if you have in mind

Reply:
Not sure whether or not. Just wanted to keep the interface agnostic for now.

Reply:
cool

Reply:
Yes, we will need to add a field `address predicate` for predicate interoperability 🎉